aiida/restapi/run_api.py

```diff
@@ -143,11 +150,16 @@ def run_api(App, Api, *args, **kwargs):
         load_dbenv()
```

Comment:
Check Removed! Actually, load_dbenv has an is_dbenv_loaded() check inside, which if not matched throws an exception.